.github/workflows/test-gh.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         minikube start --driver=docker
         eval $(minikube docker-env --shell=bash)
 
+        export SECRETGEN-CONTROLLER=true
```

Comment:
i think we should prefix this env var with KAPPCTRL_E2E_. lets also use _ instead of -

Reply:
Thanks, added